config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+   resources :keywords, only: [:index, :create]
```

Comment:
How about using `%i` here? related to [our Ruby convention](https://nimblehq.co/compass/development/code-conventions/ruby/#convention-26)

```suggestion
   resources :keywords, only: %i[:index :create]
```

Reply:
Yup, made the change 😄 Updated here https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/40f76b1ebffdddf0f3a49ba6f8c7d370bf2afcad